packages/react/src/hooks/use-aria-conditional-ids.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react';
+
+interface ConditionalValue {
+    id: string | undefined;
+    condition?: boolean;
```

Comment:
Je suis pas sur du mot condition, ca manque de precision, condition a faire quoi? P-t plutot opter pour `enabled` ou `active` ou `included`

Reply:
Je manque d'inspiration... `included`? `includeCondition`?

Reply:
On peut y aller avec `includeCondition`, pas besoin d'etre parfait.

Reply:
`include` it is! C'est affirmative et ça communique mieux l'intention!